GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -698,8 +705,14 @@ private void writeAnswer(
 
         int col = startColumn;
         for (String cellValue : cells) {
-            if (questionType == QuestionType.NUMBER) {
+        	if (questionType == QuestionType.NUMBER) {
```

Comment:
Seems your environment is using tabs instead of spaces for formatting. Can you check and change this?